consensus/src/step_votes_reg.rs

```diff
@@ -176,17 +176,21 @@ impl AttInfoRegistry {
         quorum_reached: bool,
         generator: &PublicKeyBytes,
     ) -> Option<Message> {
-        let att = self
-            .att_list
-            .entry(iteration)
-            .or_insert_with(|| IterationAtts::new(*generator));
-
-        let att_info = att.get_or_insert(vote);
-
-        att_info.set_sv(iteration, sv, step, quorum_reached);
-        att_info
-            .is_ready()
-            .then(|| Self::build_quorum_msg(&self.ru, iteration, att_info.att))
+        if let Vote::Valid(_) = vote {
```

Comment:
This way we are not aggregating invalid votes during non-emergency iterations

Reply:
Reverted. Not needed anymore.